src/D2L.CodeStyle.Analyzers/ApiUsage/DependencyInjection/Domain/ConfigurePluginsExtensionPointExpression.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace D2L.CodeStyle.Analyzers.ApiUsage.DependencyInjection.Domain {
+	// void ConfigurePlugins<TExtensionPoint, TPlugin>( ObjectScope scope )
+	//      where TExtensionPoint : IExtensionPoint<TPlugin>
+	// void ConfigureOrderedPlugins<TExtensionPoint, TPlugin, TComparer>( ObjectScope scope )
+	//      where TExtensionPoint : IExtensionPoint<TPlugin>
+	//		where TComparer : IComparer<TPlugin>, new()
+	// 	void RegisterPluginExtensionPoint<TExtensionPoint, T>()
+	//      where TExtensionPoint : IExtensionPoint<T>

```

Comment:
Planning to rename this one to the first `ConfigurePlugins`, to align with non-extensionpoints